proto/manifest.proto

```diff
@@ -12,22 +12,26 @@ message Resource {
     // Path specifies the path from the bundle root. If more than one
     // path is present, the entry may represent a hardlink, rather than using
     // a link target. The path format is operating system specific.
+    // Path MAY be an absolute path, but it MUST be interpreted with the bundle root.
```

Comment:
except symlink, right? (e.g. some executable can be a symlink to `/bin/true` or `/bin/false` on the absolute root)